### PR TITLE
update the "Manual" installation instruction on stable 

### LINKED ISSUE #2200: update the "Manual" installation instruction on stable
This is the current released docu https://mss.readthedocs.io/en/stable/installation.html

We need a backport from develop https://github.com/Open-MSS/MSS/blob/develop/docs/installation.rst on stable
because some of the links on our released installation page are broken.

### PR DESCRIPTION
**Pull request by Preetam-Das26**
**Purpose of PR?**:
update the "Manual" installation instruction on stable 
Fixes #2200 

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Does this PR results in some Documentation changes?**
_If yes, include the list of Documentation changes_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (Non-API breaking changes that adds functionality)
- [ ] PR Title follows the convention of  `<type>: <subject>`
- [ ] Commit has unit tests

### CODE REVIEW

**Comment by Preetam-Das26** [comment]
Last pull request had to be closed due to some issues with the branch and forked repo. Kindly review this new PR @ReimarBauer

File: docs/installation.rst [hunk 1]
```diff
@@ -70,7 +74,7 @@ We suggest to create a mss user.
 * login as mss user
 * create a *src* directory in /home/mss
 * cd src
-* get `mambaforge <https://github.com/conda-forge/miniforge#mambaforge>`__
+* get `miniforge <https://github.com/conda-forge/miniforge#download>`__
 * set execute bit on install script
```
Comment (ReimarBauer) [inline docs/installation.rst:78]: we should write miniforge, Miniforge  same way. When the word is used we should better write it exactly how they use it currently. Can you change it to Miniforge3.

The problem here is, that the software is used by people not heard about python and have not installed something before. They may match the word on the linked page. 

 
I can tell stories what had gone wrong in the past.

Comment (Preetam-Das26) [reply docs/installation.rst:78]: Ok sure.

**Comment by Preetam-Das26** [comment]
Changes done @ReimarBauer

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**